templates/build/agenda-base.build.hbs

```diff
@@ -30,6 +30,12 @@
   \{{/if}}
   <!-- End of Filter Overlay -->
 
+  \{{#if showAddEntry}}
+  <div class="dynamic-list-add-item focus-outline" tabindex="0">
```

Comment:
Why did the location of this `.dynamic-list-add-item` DOM element need to be moved? We have existing CSS selectors in this component and in the theme SCSS file that relies on the current structure.

Reply:
The structure of the page shows that the button for creating a new record is in the foreground and its selection should be earlier than the elements below it.  Or I'm wrong?

Reply:
I see, so you're moving this to make sure the order of selection puts the **Add** button before the list items. I think that's a good consideration but we don't need to do that for now.

Reply:
Ok, i can revert this

Reply:
Yes please. And there's also conflict with this PR so you may want to resolve them first.